### PR TITLE
Hamburger branch

### PR DESCRIPTION
**Pull request by ReemMohamedAbdelfatah**
Add hamburger menu and solve linter errors

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -36,11 +36,70 @@ header {
   padding-left: 25px;
   padding-right: 25px;
   background-color: white;
+  align-items: center;
+}
+
```
Comment (jasonfelice) [inline style.css:41]: - [ ]  [OPTIONAL]  I highly recommend fixing this grey space. The body has some margin by default. You can fix this by giving it `margin: 0`. 😄 
image

File: style.css [hunk 2]
```diff
@@ -36,11 +36,70 @@ header {
...
+  visibility: hidden;
+  background-color: #6070ff;
+  mix-blend-mode: multiply;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  z-index: 1;
+  margin: auto;
+  transition: all 400ms ease;
+  backdrop-filter: blur(4.5px);
```
Comment (jasonfelice) [inline style.css:56]: - [ ] **\[OPTIONAL\]** The mobile menu is working perfectly. However, consider changing the style to make the text stand out.

image

File: style.css (hunk 1 above)
Comment (jasonfelice) [inline style.css:41]: - [ ]  [OPTIONAL]  I highly recommend fixing this grey space. The body has some margin by default. You can fix this by giving it `margin: 0`. 😄 
image

**Review by jasonfelice — APPROVED** [review]
Hi @Kasaija-Kenneth, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
Design is good✔️
Menu is working as intended ✔️
Followed GitHub flow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @ReemMohamedAbdelfatah   , 

Your project is complete and design is beautiful 👍 !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

## Optional suggestions

- [ Optional] Please try to address the previous code reviewer Suggestion.  

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**